DHI.Mike1D.Examples/Scripts/WaterVolumeChangeDA.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using DHI.Generic.MikeZero;
+using DHI.Mike1D.Engine;
+using DHI.Mike1D.Engine.ModuleData;
+using DHI.Mike1D.Engine.ModuleHD;
+using DHI.Mike1D.Generic;
+using DHI.Mike1D.Mike1DDataAccess;
+using DHI.Mike1D.Plugins;
+using DHI.Mike1D.ResultDataAccess;
+
+
+namespace DHI.Mike1D.Examples.Scripts
+{
+  public class WaterVolumeChangeDA
+  {
+    private Mike1DController _mike1dController;
+    private EngineNet _engineNet;
+    private DAWaterVolumeChangeCalculator _daWaterVolumeChangeCalculator;
+
+    /// <summary>
+    /// Method called when IMike1DController is available.
+    /// </summary>
+    [Script]
+    public void Initialize(IMike1DController controller)
+    {
+      controller.ControllerEvent += ControllerOnControllerEvent;
+    }
+
+
+    private void ControllerOnControllerEvent(object sender, ControllerEventArgs e)
+    {
+      if (e.State == ControllerState.Initialized)
+      {
+        _mike1dController = (Mike1DController)sender;
+        _mike1dController.Mike1DData.ResultSpecifications[0].WhatAddUnique(new Quantity("WaterVolumeChangeDA", "DA - Water volume change", eumItem.eumIVolume));
```

Comment:
I can see that WaterVolumeChangeDA quantity is used a few times. Does it make sense to extract it into an instance variable and reuse?